ecc/chaincode/enclave/enclave.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 // #cgo CFLAGS: -I${SRCDIR}/ecc-enclave-include -I${SRCDIR}/../../../common/sgxcclib
-// #cgo LDFLAGS: -L${SRCDIR}/ecc-enclave-lib -lsgxcc
+// #cgo LDFLAGS: -L${SRCDIR}/ecc-enclave-lib -lsgxcc -lgcov
```

Comment:
is this necessary to to compile ecc now?

Reply:
If code coverage is enabled, then yes, because the libraries will contain gcov symbols, so the final executable must be linked accordingly.

If code coverage is disabled, then no, and in this case the compiler is intelligent enough not to add any gcov-related thing to the binary.